bigquery-storage/api/BigQueryStorage.Samples/BigQueryStorage.Samples.csproj

```diff
@@ -4,7 +4,8 @@
     <IsPackable>false</IsPackable>
   </PropertyGroup> 
   <ItemGroup>
+    <PackageReference Include="Apache.Avro" Version="1.11.1" />
     <PackageReference Include="Google.Cloud.BigQuery.Storage.V1" Version="3.6.0" />
-    <PackageReference Include="Google.Protobuf" Version="3.22.0" />  
+    <PackageReference Include="Google.Protobuf" Version="3.22.0" />
```

Comment:
Can you check if we need this? I'd say we don't as we don't as we probably already updated GAX to the protobuf version we needed?
Just remove it and if everything builds, we are fine.